apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -205,6 +215,29 @@ export default connect(
       }
     }
 
+    displayErrorMsgs() {
+      if (this.props.logOutput.size > 0) {
+        return this.props.logOutput.map(errorMsg => {
+          if (typeof errorMsg === 'object') {
+            errorMsg = errorMsg.toJS();
+          }
+          return (
+            <div>
```

Comment:
Does each Inspector element need to be wrapped in `<div>` tags?

Reply:
Ah! Yes, there's no need for it to be wrapped in `<div>`. Based on the example given from the documentation I thought it had to be wrapped in the div tag but nope! I went ahead and removed it then also added the key prop to the` <Inspector>`.

Result would be: `<Inspector key={i} data={errorMsg} />`